chainstate/tx-verifier/src/transaction_verifier/signature_destination_getter.rs

```diff
@@ -118,17 +128,37 @@ impl<'a> SignatureDestinationGetter<'a> {
                                     .decommission_destination()
                                     .clone())
                             }
+                            TxOutput::TokensOp(v) => match v {
+                                TokenOutput::MintTokens(_, _, d) => Ok(d.clone()),
+                                TokenOutput::IssueNft(_, _, d) => Ok(d.clone()),
+                                TokenOutput::IssueFungibleToken(_)
+                                | TokenOutput::RedeemTokens(_, _)
+                                | TokenOutput::LockCirculatingSupply(_) => {
+                                    // This error is emitted in other places for attempting to make this spend,
+                                    // but this is just a double-check.
+                                    Err(SignatureDestinationGetterError::SigVerifyOfBurnedOutput)
```

Comment:
The error label is incorrect.

Reply:
I would call it SigVerifyOfUnspendableOutput? Is there a better label?

Reply:
renamed to `SigVerifyOfNotSpendableOutput`